pkgs/racket-doc/scribblings/reference/procedures.scrbl

```diff
@@ -904,5 +904,20 @@ arguments that procedures with arity @racket[b] accept.
 
 }
 
+@defproc[(get-failure-result [v failure-result/c]) any]{
+
+If @racket[v] is a procedure, it is applied in @tech{tail position} to zero
+arguments to produce a result. Otherwise, @racket[v] is returned.
+
+@racket[get-failure-result] can be used to extract the value of failure result
```

Comment:
Nit: should be "The `get-failure-result` function" to avoid starting a sentence with a lowercase identifier (per the style guide).

Reply:
I’ve reworded as suggested.